integration/src/main/java/scripts/testscripts/ControlledGcsBucketLifecycle.java

```diff
@@ -258,7 +274,10 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
     assertEquals(1, bucketList.getResources().size());
     MultiResourcesUtils.assertResourceType(ResourceType.GCS_BUCKET, bucketList);
 
-    // Owner can delete the bucket through WSM
+    // -- Test Case 6 --
```

Comment:
Not specific to this review - It is troubling that we have 6 cases in this one test. I can see why for these integration tests where they're testing the whole lifecycle, but I can also see why this would take a long time. Especially if we have to wait between cases.